webapi/src/main/scala/org/knora/webapi/responders/admin/UsersResponderADM.scala

```diff
@@ -1420,34 +1403,50 @@ class UsersResponderADM(responderData: ResponderData) extends Responder(responde
       * Helper method for checking if an username is already registered.
       *
       * @param username the username of the user.
+      * @param current the current username of the user.
       * @return a [[Boolean]].
       */
-    private def userByUsernameExists(username: String): Future[Boolean] = {
-        for {
-            askString <- Future(queries.sparql.admin.txt.checkUserExistsByUsername(username = username).toString)
-            // _ = log.debug("userExists - query: {}", askString)
-
-            checkUserExistsResponse <- (storeManager ? SparqlAskRequest(askString)).mapTo[SparqlAskResponse]
-            result = checkUserExistsResponse.result
-
-        } yield result
+    private def userByUsernameExists(username: String, current: String = ""): Future[Boolean] = {
```

Comment:
Please don't use `""` to represent a value that does not exist. That's why Scala has the `Option` class. Use `None` instead:

```scala
private def userByUsernameExists(username: String, current: Option[String] = None)
```